sql/moz-fx-data-shared-prod/fenix_derived/attributable_clients_v1/query.sql

```diff
@@ -109,21 +103,18 @@ SELECT
     ELSE 0
   END AS ad_clicks,
   COALESCE(activated, FALSE) AS activated,
+  COALESCE(activated AND activation_date = submission_date, FALSE) AS activated_on_this_day,
```

Comment:
This may be controversial. I would like the ability to partition on activated users overall. This is still imperfect (first 6 days won't say whether they're activated), but in this way filtering on `activated` will give e.g. DAU for activated users, not the number of activations. For # of activations, we'd look at `activated_on_this_day`

Reply:
Yes, I think `activated_on_this_day` is important to have on the table. I still find it easier to understand the table and write queries when the things we want to count are metrics in a table designed for them to be [additive](https://www.kimballgroup.com/data-warehouse-business-intelligence-resources/kimball-techniques/dimensional-modeling-techniques/additive-semi-additive-non-additive-fact/), and this is a step in that direction if we don't want to actually have it as a number.

I'm not entirely sure we need `activated`. It's not one of our official segments and only once we performed an analysis on how activated users compare to non-activated ones. Do you see that request re-ocurring often enough to justify having a hard to understand table with the two different activated values?

Reply:
I guess I think of `activated` as a dimension, not a fact, especially in regards to this table. Maybe we have two fields:
- `is_activated`: Whether the user is activated, a dimension
- `activations_count`: Number of activations, a fact (and additive)

Looking at `is_actiated` across a number of users was kind of the whole reason I started this PR 😄 where as we're trying to understand how our activated users behave, it makes more sense to target them in certain ways (e.g. marketing)

Reply:
Yes, that would be perfect.